flink-libraries/flink-table/src/main/scala/org/apache/flink/table/TableAPIConfigConstant.scala

```diff
@@ -0,0 +1,26 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.flink.table
+
+/**
+  * All configuration constants
+  */
+object TableAPIConfigConstant {
+  // used by event-time over window,
+  val OVER_EVENT_TIME_ALLOWED_LATENESS = "OVER_EVENT_TIME_ALLOWED_LATENESS"
```

Comment:
I think we should not introduce allowed lateness yet. There are several other operators that would need this parameter and we should add it either to all or to none. 
I haven't looked at the semantics yet, but in the DataStream API, allowed lateness will trigger a recomputation, i.e., it would sent out updated results. For the Table API, we will need support for retraction before we can do the same.